demo/index.html

```diff
@@ -11,13 +11,13 @@
 <html>
 <head>
 
-  <title>core-pages</title>
+  <title>iron-pages</title>
 
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Use this meta tag (magical invocation for HW acceleration on Android): `<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">`